rclcpp/src/rclcpp/callback_group.cpp

```diff
@@ -56,6 +56,16 @@ CallbackGroup::type() const
   return type_;
 }
 
+size_t
+CallbackGroup::size() const
+{
+  return subscription_ptrs_.size() +
+         service_ptrs_.size() +
+         client_ptrs_.size() +
+         timer_ptrs_.size() +
+         waitable_ptrs_.size();
```

Comment:
nitpick: vertical aligning indents are generally avoided, however in this case since the length of `return ` will never change so the indent shouldn't either, so the main argument for it isn't valid here. I would have still used `()` to get them on a single two space indent.

not blocking